docs/griptape-framework/structures/src/json_schema_rule.py

```diff
@@ -0,0 +1,14 @@
+import schema
+
+from griptape.rules.json_schema_rule import JsonSchemaRule
+from griptape.structures import Agent
+
+agent = Agent(
+    rules=[
+        JsonSchemaRule(
+            schema.Schema({"answer": str, "relevant_emojis": schema.Schema(["str"])}).json_schema("Output Format")
```

Comment:
Would be cool to give an example using pydantic model to create the schema, then converting the output text back into a python object. I can think of some people in particular who might appreciate such an example. I suppose it doesn't need to be pydantic, but showing how to get a python object back seems like a good way to highlight the power of this new rule.

(Like tada, now you can use autocomplete in your editor again!)